StreamChatClient_v3/Utils/Result+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension Result {
```

Comment:
Let's not make this extension public. It can conflict with some extension our customers already have.